prisma/schema.prisma

```diff
@@ -34,11 +34,17 @@ model Experiences {
 
 model Places {
   id           Int      @id @default(autoincrement())
-  place_type   PlaceType     @default(ONSEN) // NOTE Needs to be more strict
+  place_type   PlaceType     @default(ONSEN)
   name         String   @db.VarChar(255)
   address      String   @db.VarChar(255)
   latitude     Decimal @db.Decimal(9, 6)
   longitude    Decimal @db.Decimal(9, 6)
+  google_place_id String? @unique @db.VarChar(255)
+  google_map_uri String? @db.Text
+  website_uri String? @db.Text
+  regular_opening_hours String? @db.Text
```

Comment:
places.regular_opening hours would be an string? I remember that is a whole object in maps

Its ok to try to added it, but its not ok to only stored as a JSON file.

Check how the structure is and how we can added into our schema